### PR TITLE
Renamed type for better searchability/docs

### PR DESCRIPTION
**Pull request by Eilon**
Also added doc comments.

@rynowak

### CODE REVIEW

File: src/Microsoft.Framework.TelemetryAdapter/TelemetrySourceAdapterServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -6,8 +6,13 @@
 
 namespace Microsoft.Framework.DependencyInjection
 {
-    public static class ServiceCollectionExtensions
+    public static class TelemetrySourceAdapterServiceCollectionExtensions
     {
+        /// <summary>
+        /// Extension methods for setting up telemetry source adapter related services in an <see cref="IServiceCollection" />.
```
Comment (rynowak) [inline src/Microsoft.Framework.TelemetryAdapter/TelemetrySourceAdapterServiceCollectionExtensions.cs]: > 120 chars :astonished:

**Comment by rynowak** [comment]
:shipit:

File: src/Microsoft.Framework.TelemetryAdapter/TelemetrySourceAdapterServiceCollectionExtensions.cs (hunk 1 above)
Comment (Eilon) [inline src/Microsoft.Framework.TelemetryAdapter/TelemetrySourceAdapterServiceCollectionExtensions.cs]: Meh!

Comment (avanderhoorn) [inline src/Microsoft.Framework.TelemetryAdapter/TelemetrySourceAdapterServiceCollectionExtensions.cs]: Thanks how @Eilon rolls :sunglasses:

Comment (Eilon) [inline src/Microsoft.Framework.TelemetryAdapter/TelemetrySourceAdapterServiceCollectionExtensions.cs]: :+1:

**Pull request merged**